src/client/components/Footer/index.js

```diff
@@ -29,43 +27,43 @@ class Footer extends React.PureComponent {
           </Nav>
           <Nav>
             <Nav.Link href="https://www.facebook.com/californiaedd/">
-              <IcomoonReact
-                iconSet={iconSet}
-                color="#ffffff"
-                size={20}
-                icon="facebook"
+              <img
+                src="images/facebook.svg"
+                width="15"
+                height="15"
+                alt="facebook icon"
```

Comment:
Should the alt text be localized?

Reply:
I got out of practice when we put all the content in for the launch! Thanks for catching this. Updating now!